packages/commonwealth/client/scripts/views/pages/overview/index.tsx

```diff
@@ -0,0 +1,100 @@
+/* @jsx m */
+
+import m from 'mithril';
+
+import 'pages/overview/index.scss';
+
+import app from 'state';
+import { navigateToSubpage } from 'app';
+import { LoadingRow } from '../../components/loading_row';
+import { CWText } from '../../components/component_kit/cw_text';
+import { CWButton } from '../../components/component_kit/cw_button';
+import { TopicSummaryRow } from './topic_summary_row';
+import { isWindowExtraSmall } from '../../components/component_kit/helpers';
+import { CWIconButton } from '../../components/component_kit/cw_icon_button';
+import { CWDivider } from '../../components/component_kit/cw_divider';
+import Sublayout from '../../sublayout';
+
+class OverviewPage implements m.ClassComponent {
+  private initializing: boolean;
+
+  oninit() {
+    app.recentActivity.getRecentTopicActivity().then(() => {
+      m.redraw();
+    });
+  }
+
+  view() {
+    if (this.initializing) {
```

Comment:
This doesn't seem to be initializing anywhere? or waiting for the initialization (oninit) to finish before redrawing

oninit method needs to first set this.initializing to true and then upon response set it equal to false before a redraw for this to work. Otherwise, on hard refresh, there won't be any topic activity at page render to render